config/connection.js

```diff
@@ -1,15 +1,18 @@
-require('dotenv').config();
-
 const Sequelize = require('sequelize');
+require('dotenv').config();
 
 const sequelize = process.env.JAWSDB_URL
   ? new Sequelize(process.env.JAWSDB_URL)
-  : new Sequelize(process.env.DB_NAME, process.env.DB_USER, process.env.DB_PASSWORD, {
+  : new Sequelize(
+    process.env.DB_NAME,
+    process.env.DB_USER,
+    process.env.DB_PASSWORD, {
       host: 'localhost',
       dialect: 'mysql',
       port: 3306,
       dialectOptions: {
         decimalNumbers: true,
+        socketPath: process.env.SOCKET_PATH || "/var/mysql/mysql.sock"
```

Comment:
@Reyes-Jose , this default path needs to be tested. I've manually provided the SOCKET_PATH in the .env file since my default path appears to be different than other's path. Could you check this for me? I recall your default path being `mysql`